### PR TITLE
Consolidate Appointment Lists

### PR DESCRIPTION
**Pull request by narin**
## Description
Consolidate appointment lists into a single list, displaying future appointments, requests, and cancellations as well

## Testing done
Local and unit

## Screenshots
localhost_3001_vaos_appointments (1)

## Acceptance criteria
- [ ] Pending and confirmed appointments should be in the same list
- [ ] Ordering should be chronological
- [ ] AM / PM should be used in the sort order for requests
- [ ] Requests should come after any overlapping confirmed appointments

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/components/ConfirmedAppointmentListItem.jsx [hunk 1]
```diff
@@ -1,53 +1,85 @@
...
           ) : (
-            getAppointmentLocation(appointment)
+            <i className="fas fa-check-circle vads-u-color--green" />
           )}
+          <span className="vads-u-font-weight--bold vads-u-margin-left--1 vads-u-display--inline-block">
+            {canceled ? 'Canceled' : 'Confirmed'}
+          </span>
         </div>
+
+        {!canceled && (
```
Comment (jbalboni) [inline src/applications/vaos/components/ConfirmedAppointmentListItem.jsx]: I don't think we can cancel CC appointments, so might want to add that to the logic here.

**Review by jbalboni — APPROVED** [review]
Left some comments, looks good overall

**Comment by 1Copenut** [comment]
@narin && @jbalboni I pulled this branch and ran it through the axe browser plugin. It's throwing a number of errors around the definition lists not being nested properly. I took a run at a basic handling earlier today (sorry I was so late delivering this) that tests well with screen readers and validates in axe. Take what you need from this, and discard the rest. We can always polish as we move toward a design system component.

```html
<article aria-labelledby="card-1" class="vsp-card" role="region">
  <header class="vsp-card__appt-header">
    <div>
      <span class="vsp-card__confirmation" id="card-1">
        Confirmed <span class="sr-only">appointment</span>
      </span>
      <h2 class="vsp-card__heading">
        January 15, 2020 at 1:15
        <span aria-hidden="true">p.m. MST</span>
        <span class="sr-only">PM Mountain Standard Time</span>
      </h2>
      <a class="vsp-card__link" href="#">Add to calendar</a>
    </div>
    <div>
      <button>
        Cancel
        <span class="sr-only">appointment on January 15, 2020</span>
      </button>
    </div>
  </header>
  <div>
    <dl class="vsp-card__data-list">
      <dd>
        <div>
          <dl>
            <dt class="vsp-card__data-term">Community Care appointment</dt>
            <dd></dd>
          </dl>
        </div>
        <div>
          <dl>
            <dt class="vsp-card__data-term">Metro Community Health Center</dt>
            <dd>
              <p class="h-adr"><span class="p-street-address">1789 S. Braddock <span aria-hidden="true">Ave</span> <span class="sr-only">Avenue</span></span> <span class="p-extended-address">Suite 410</span> <span><span class="p-locality">Pittsburgh,</span> <span class="p-region"><span aria-hidden="true">PA</span> <span class="sr-only">Pennsylvania</span></span> <span class="p-postal-code">15218</span></span></p>
            </dd>
          </dl>
        </div>
        <div>
          <dl>
            <dt class="vsp-card__data-term">Follow Up/Routine Visit</dt>
            <dd>This is the longer explanation of the routine visit. It could be something generic or specialized.</dd>
          </dl>
        </div>
      </dd>
    </dl>
  </div>
</article>
```

**Review by rianfowler — APPROVED** [review]

**Comment by 1Copenut** [comment]
@narin I pulled your branch again and did some checking. The axe reporter is still giving me a number of errors for the definition lists. LMK if you'd like to talk tomorrow, and we can review code.

**Comment by narin** [comment]
@1Copenut Thank you for checking this out again.  I didn't see any AXE errors when I ran it recently, but since this is already merged, can you please capture what you saw in a new ticket, tag me on it, and I'll address them ASAP? Thanks!

**Pull request merged**